inventory/src/components/KitIndex/KitList.js

```diff
@@ -0,0 +1,152 @@
+import {
+  Accordion,
+  AccordionSummary,
+  Box,
+  Grid,
+  MenuItem,
+  Pagination,
+  Select,
+  Skeleton,
+  Stack,
+  TextField,
+} from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import React, { useEffect, useState } from 'react';
+
+import { LoadingSpinner } from '..';
+import { ORDERS_PER_PAGE } from '../../globals';
+import '../../globals/styles/inventoryBase.scss';
+import { useKits } from '../../hooks/queries';
+
+import { KitAdd } from './KitAdd';
+import { KitData } from './KitData';
+
+export const KitList = () => {
+  const isMobile = useMediaQuery('(max-width: 800px)');
+  const { data: kitsData, isLoading: dataLoading } = useKits();
+  const [currentPage, setCurrentPage] = useState(1);
+  const [filter, setFilter] = useState('all');
+  const [searchTerm, setSearchTerm] = useState('');
+  const [kitsToDisplay, setKitsToDisplay] = useState([]);
+  const startIndex = (currentPage - 1) * ORDERS_PER_PAGE;
+  const endIndex = startIndex + ORDERS_PER_PAGE;
+  useEffect(() => {
+    if (!kitsData) return;
+    const newKits = kitsData.kits.filter(
+      (k) =>
+        (!searchTerm ||
+          k.blueprint_name.toLowerCase().includes(searchTerm.toLowerCase())) &&
+        (filter === 'all' || k.complete === filter),
+    );
+    setKitsToDisplay(newKits);
+  }, [kitsData, searchTerm, filter]);
+
+  const handlePageChange = (_, value) => {
+    setCurrentPage(value);
+  };
+  return (
+    <Box
+      sx={{
+        width: 1,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+      }}
+    >
+      <Box
+        className='dynamic-width'
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          marginBottom: 2,
+          justifyContent: 'space-between',
+          gap: 3,
+        }}
+      >
+        <Box sx={{ display: 'flex', width: 1 }}>
+          <TextField
+            label='Search by Blueprint'
+            value={searchTerm}
+            onChange={(e) => setSearchTerm(e.target.value)}
+            sx={{ width: 1 }}
+          />
```

Comment:
Modify search

* Add filters (similar to items index) for kit statuses (READY, LOANED) and completeness (COMPLETE, INCOMPLETE)
* Change placeholder to "Search by.."
* In select field, allow users to search by "Kit Name" or "Blueprint Name"